.gitignore

```diff
@@ -0,0 +1,28 @@
+Makefile
+Makefile.in
+aclocal.m4
+m4/*.m4
+autom4te.cache/
+config/
+configure
+config.log
+config.status
+*/Makefile.in
+*/Makefile
+include/config.in
+include/config.h
+include/stamp-h1
+*/.deps/
+libtool
+.libs
+*.o
+*.lo
+*.la
+.dirstamp
+cdrwtool/cdrwtool
+mkudffs/mkudffs
+pktsetup/pktsetup
+udffsck/udffsck
+udfinfo/udfinfo
+udflabel/udflabel
+wrudf/wrudf
```

Comment:
I'm not a big fan of gitignore, it just hide if development git directory is clean or not, forces to use git clean -x (ignore gitignore). Also specially if people add there their own files which do not exists on other computers... Anyway everybody can sets its own set of gitignore rules (e.g. via file info/exclude) locally.